index.html

```diff
@@ -17,25 +17,24 @@
       <div class="logo">
         <a href="#" id="name">Nimrod Acosta</a>
       </div>
-      <button class="normalb">
-        <img src="./Images/Union.png" class="union" alt="hamburger menu" />
-      </button>
-      <nav class="burger">
-        <a href="#">
-          <p class="headermenu">Portfolio</p>
+      <div class="burger" onclick="toggleMenu();">       
+      </div>
+      <nav class="menubar">
```

Comment:
- [ ] The mobile menu is displayed at the page load, please consider changing this so it will be displayed only when clicking on the hamburger menu